README.md

```diff
@@ -1,7 +1,8 @@
-### Wikipedia Android app
+### Pen Pineapple Apple Pen Wikipedia Android app
```

Comment:
We've actually already updated the readme to include the codacy badge and a link to our actual wiki, so this will likely conflict when we merge develop onto master.